lti_consumer/lti_1p3/README.md

```diff
@@ -100,13 +100,13 @@ def lti_launch_endpoint(request):
 
     # Required user claim data
     lti_consumer.set_user_data(
-      user_id=request.user,
+      user_id=request.user.id,
```

Comment:
I'm just updating the README with changes I had to make when borrowing this code.